company/okrs/2020_q1.md

```diff
@@ -0,0 +1,84 @@
+## Product
+
+1. **Product: WIP Deliver on product roadmap**
+   1. Product KR: Roadmap communicates rough timeline for upcoming work.
+   1. Product KR: Build out per-team strategy RFCs to buffer roadmap.
+   1. Product KR: Dogfooding each feature release to ensure quality UX.
+1. **Product: WIP Reduce product friction**
+   1. Product KR: Reduce time to "aha" (0 to search).
+   1. Product KR: Reduce time 0 to deploy.
+   1. Product KR: Improve out of the box experience.
+1. **Product: WIP Grow product team**
+   1. Product KR: Hire to fill position gaps (1 UX Designer, 1 PM).
+   1. Product KR: Create handbook section outlining product strategy and operation guide.
+   1. Product KR: Create scalable customer feature prioritization.
+
+## Engineering
+
+1. **VP Eng: Build a world class engineering team.**
+   1. VP Eng KR: Grow team to plan (+1 code intel, +2 distribution, +2 web, +1 core services).
+   1. VP Eng KR: Transition an existing teammate to be (or hire) an Engineering Manager of the distribution team.
+   1. VP Eng KR: Document how we do performance reviews and determine compensation for teammates in engineering roles.
+1. **VP Eng: Create a data driven planning process.**
+   1. VP Eng KR: Each item in a tracking issue has an estimate of time or size.
+   1. VP Eng KR: Each team commits to a written work plan for each release cycle that accounts for the work estimates and the team's capacity.
+   1. VP Eng KR: Each team completes >80% of the work that they committed to at the beginning of the release cycle each iteration.
+
+1. **Distribution: Resolve issues for critical customer deals and expansions**
+1. **Distribution: Improve developer productivity**
+   1. Distribution KR: Lower build times to X
+   1. Distribution KR: Automate away release captain role
+   1. Distribution KR: Infrastructure for automated deploys and releases (no human in loop required by default)
+1. **Distribution: Build features that accelerate customers through pipeline**
+   1. Distribution KR: Shorten the time from "intent to try Sourcegraph" to "whole org is using Sourcegraph" (onboarding experience)
+   1. Distribution KR: Ensure new features (LSIF, GitLab Native Integration, Automation) are accessible and easily tryable by customers once they go GA
+   1. Distribution KR: Surface information to site admins and managers to encourage expansion/upsell
+
+1. **Code intelligence: Make it easy for customers to set up LSIF**
+   1. Code intelligence KR: At least three customers are using LSIF on their primary repositories
+   1. Code intelligence KR: At least one customer is using LSIF across all of their code in languages for which there exists an LSIF indexer
+
+1. **Code intelligence: Make it clear where code intel results are coming from**
+   1. Code intelligence KR: Code intelligence UI (tooltips, references panel, etc.) indicates whether or not results are precise
+   
+1. **Code intelligence: Increase precision and recall of LSIF for nearest-commit**
```

Comment:
@chrismwendt This is a good objective statement on its own, but now there are 4 objectives and it would be nice to stick to no more than 3. Also, the "Make it clear where code intel results are coming from" objective above doesn't feel meaty enough to be its own objective.

What if we combined these objectives (leave KRs unchanged) and call the new objective something like "Increase user confidence in code intelligence provided by LSIF"?